### ISSUE TITLE
AutoUpload error

### ISSUE THREAD

**Issue by TheSimpleWriter** [nextcloud/android#7543]
### Steps to reproduce
1. Have folder of photos to upload
2. Add that folder with photos to AutoUpload with custom filder to upload to
3. Wait until it crashes

### Expected behaviour
- it should just upload photos

### Actual behaviour
- app crashes after some period of time

### Can you reproduce this problem on https://try.nextcloud.com?
- Yes, I can reproduce in either mine and that server

### Environment data
Android version:
10
Device model: 
Xiaomi mi 9 se
Stock or customized system:
Stock
Nextcloud app version:
3.14
Nextcloud server version:
19
Reverse proxy:
No

Brand: Xiaomi
Device: grus
Model: Mi 9 SE
Id: QKQ1.190828.002
Product: grus

SDK: 29
Release: 10
Incremental: V12.0.2.0.QFBMIXM

ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

#### Web server error log
```
android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{7e87992 u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1973)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7562)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)

```

**Comment by TheSimpleWriter** [comment]
Sorry if I filled up the form incorrectly

**Comment by tobiasKaminsky** [comment]
Will be fixed with 3.14.1, which is already available as a test version via play store beta program.
